ecology.yaml

```diff
@@ -0,0 +1,525 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: 'Statistics'
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: 'Annotation'
+
+  - name: add_input_name_as_column
+    owner: mvdbeek
+    tool_panel_section_label: 'Text Manipulation'
+ 
+  - name: regex_replace
+    owner: kellrott
+    tool_panel_section_label: 'Text Manipulation'
+
+  - name: find_in_reference
```

Comment:
I think this one can also replaced by already existing tools?

Reply:
Already on [usegalaxy.eu](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/jjohnson/find_in_reference/find_in_reference/0.0.3) so it is ok to keep this one?

Reply:
Right! So I remove it from this list for now.